src/main/java/sirius/biz/tenants/jdbc/SQLTenantExportJobFactory.java

```diff
@@ -43,5 +44,12 @@ protected void extendSelectQuery(SmartQuery<SQLTenant> query, ProcessContext pro
             query.eq(SQLTenant.TENANT_DATA.inner(TenantData.PACKAGE_DATA).inner(PackageData.PACKAGE_STRING),
                      packageName);
         });
+
+        processContext.getParameter(upgradesParameter).ifPresent(upgrades -> {
+            query.where(query.filters()
+                             .allInField(SQLTenant.TENANT_DATA.inner(TenantData.PACKAGE_DATA)
+                                                              .inner(PackageData.UPGRADES),
+                                         List.of(upgrades.split(","))));
```

Comment:
**Question:** Aren't there CSV-Filters that allow to directly pass in a comma separated string to filter on?